airflow/providers/google/cloud/operators/automl.py

```diff
@@ -52,12 +55,36 @@
 MetaData = Sequence[Tuple[str, str]]
 
 
+def _raise_exception_for_deprecated_operator(
+    deprecated_class_name: str, alternative_class_names: str | list[str]
+):
+    if isinstance(alternative_class_names, str):
+        alternative_class_name_str = alternative_class_names
+    elif len(alternative_class_names) == 1:
+        alternative_class_name_str = alternative_class_names[0]
+    else:
+        alternative_class_name_str = ", ".join(f"`{cls_name}`" for cls_name in alternative_class_names[:-1])
+        alternative_class_name_str += f" or `{alternative_class_names[-1]}`"
+
+    raise AirflowProviderDeprecationWarning(
+        f"{deprecated_class_name} for text, image and video prediction is "
+        f"deprecated and will be removed after 31.03.2024. All the functionality of "
```

Comment:
Please update similar warnings as well, as it has already been removed by GCP:

```suggestion
        f"{deprecated_class_name} for text, image, and video prediction has been "
        f"deprecated and no longer available. All the functionality of "
```